src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -126,7 +161,7 @@ public override Task SendGroupAsync(string groupName, string methodName, object[
             if (group != null)
             {
                 var message = CreateInvocationMessage(methodName, args);
-                var tasks = group.Values.Select(c => SafeWriteAsync(c, message));
+                var tasks = group.Values.Select(c => c.WriteAsync(message));
```

Comment:
Write a helper method for calling WriteAsync on a group of connection contexts? Can avoid the overhead of LINQ and closure.

Reply:
I'm not fixing LINQ in this change. We need to add benchmarks for groups and the other overloads. I almost went down this rabbit hole but resisted as part of this change.